encoders/image/CLIPImageEncoder/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM jinaai/jina
```

Comment:
FROM jinaai/jina

# install git
RUN apt-get -y update && apt-get install -y git

# setup the workspace
COPY . /workspace
WORKDIR /workspace

# install the third-party requirements
RUN pip install git+https://github.com/openai/CLIP.git

ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]